src/components/calculator/SelectControl.tsx

```diff
@@ -1,16 +1,31 @@
+import num2fraction from 'num2fraction'
 import React from 'react'
 import { Badge, OverlayTrigger } from 'react-bootstrap'
 
 import { CalculatorData } from 'data/calculate'
 import { FormValue } from 'data/data'
 
+function showRiskMultiplier(multiplier: number): string {
+  // console.log(multiplier)
+  // console.log(multiplier < 0)
+  // debugger
+  if (multiplier === 1) {
+    return 'default risk'
+  } else if (multiplier > 0 && multiplier < 1) {
+    return `${num2fraction(multiplier)}th the risk`
```

Comment:
Could you add custom text for the "Type of Interaction" dropdown? This text doesn't reflect what that dropdown does. It should be:

One time [6% per hour]
Housemate [30% each week]
Spouse / Partner [48% each week]

Reply:
Smart. I like it. I'll just hard code that in.